include/RAJA/policy/openmp/policy.hpp

```diff
@@ -55,52 +86,58 @@ struct Static : std::integral_constant<unsigned int, ChunkSize> {
 //////////////////////////////////////////////////////////////////////
 //
 
+namespace exec {
```

Comment:
I placed all execution policy names in the exec namespace so users can autocomplete `RAJA::policy::omp::` to get the various schedules and `default_chunk_size`